frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -35,7 +35,7 @@
       <div v-show="searchResults.length !== 0 && !showLoader"
            class="notification is-large is-unselectable has-text-centered is-clickable py-1 mb-1"
            @mousedown="globalSearch()">
-        Limited to 50 results per type. Click here to search all integrated GEMs
+        Limited to 50 results per type. Click <span class="pseudoLink"> here </span> to search all integrated GEMs
```

Comment:
I was imagining the highlight to be applied differently, which actually surfaced a bug.

This line of text should show only when there are search results, but due to how the object `searchRsults` looks like, the line of text always shows, even if there are no results. So the condition here should be refactored to only be shown if there truly are results.

There is a place lower down in this file where `{{ messages.searchNoResult }}` is shown when there are no results. That is the place where we want our users to be encouraged to click so that they search the other models.

Reply:
It's a bit weird that I don't see any effect of highlighting. 
<img width="901" alt="Screenshot 2021-10-04 at 16 09 56" src="https://user-images.githubusercontent.com/5833722/135866899-7bd82608-ce7b-4107-b254-9e2ff1f32f43.png">

<img width="829" alt="Screenshot 2021-10-04 at 16 10 28" src="https://user-images.githubusercontent.com/5833722/135866917-3822875c-273d-45a4-a482-38ea34b70154.png">

Reply:
Please have a look at aad35c954984c9715d59506fd691a59bd21a53e2 and see if that is what you are after

Reply:
This is so much better @inghylt 

<img width="802" alt="image" src="https://user-images.githubusercontent.com/23480589/135923096-17963bfa-cd5b-499a-8a1b-4484e7268ca0.png">

Could you make the entire area clickable, instead of just the word `here`?

Reply:
Sure. Do you want to keep `here` styled as a link, the entire message styled as a link or no link styling at all?

Reply:
Whatever you think is best UI-wise. As long as the user doesn't have to chase a small word to click on, I'm happy.

Reply:
I'm also fine with the whatever UI you think the best.